<ui version="4.0" >
 <class>LaAssemblageConversionBase</class>
 <widget class="QDialog" name="LaAssemblageConversionBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>443</width>
    <height>393</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Assemblage Conversion Utility</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="../resources/resources.qrc" >:/la_icon_small.png</iconset>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" colspan="4" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>Enter the NISP and Kg usable meat for each animal in the model</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="4" >
    <widget class="QTableWidget" name="tblAnimals" >
     <property name="toolTip" >
      <string/>
     </property>
     <property name="whatsThis" >
      <string/>
     </property>
     <property name="alternatingRowColors" >
      <bool>true</bool>
     </property>
     <property name="selectionMode" >
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior" >
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="showGrid" >
      <bool>false</bool>
     </property>
     <column>
      <property name="text" >
       <string>Animal</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Number</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Usable Meat</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Cals/Kg</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>% of Diet</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="2" column="0" colspan="4" >
    <widget class="QFrame" name="frame" >
     <property name="minimumSize" >
      <size>
       <width>400</width>
       <height>70</height>
      </size>
     </property>
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <layout class="QHBoxLayout" >
        <item>
         <layout class="QGridLayout" >
          <item row="0" column="0" >
           <widget class="QRadioButton" name="rbManual" >
            <property name="text" >
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="1" >
           <widget class="QLineEdit" name="leAnimal" >
            <property name="toolTip" >
             <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Animal Name&lt;/p>&lt;/body>&lt;/html></string>
            </property>
           </widget>
          </item>
          <item row="0" column="2" >
           <widget class="QSpinBox" name="sbUsableMeat" />
          </item>
          <item row="0" column="3" >
           <widget class="QSpinBox" name="sbCalsPerKg" />
          </item>
          <item row="1" column="0" colspan="4" >
           <widget class="Line" name="line" >
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="2" column="0" >
           <widget class="QRadioButton" name="rbAuto" >
            <property name="text" >
             <string/>
            </property>
           </widget>
          </item>
          <item row="2" column="1" colspan="3" >
           <widget class="QComboBox" name="cboAnimal" />
          </item>
          <item rowspan="3" row="0" column="4" >
           <widget class="Line" name="line_4" >
            <property name="orientation" >
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QGridLayout" >
          <item row="0" column="0" >
           <widget class="QDoubleSpinBox" name="dsbNumber" />
          </item>
          <item rowspan="2" row="0" column="1" >
           <widget class="Line" name="line_2" >
            <property name="lineWidth" >
             <number>3</number>
            </property>
            <property name="orientation" >
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
          <item rowspan="2" row="0" column="2" >
           <widget class="QPushButton" name="pbnInsert" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Preferred" hsizetype="Fixed" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize" >
             <size>
              <width>50</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize" >
             <size>
              <width>60</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text" >
             <string>Insert</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" >
           <widget class="QComboBox" name="cboNumber" >
            <item>
             <property name="text" >
              <string>NISP</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>MNI</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>Kg Bone</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" >
    <widget class="QPushButton" name="pbnCalculate_2" >
     <property name="text" >
      <string>Calculate</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" >
    <widget class="QPushButton" name="pbnCalculate" >
     <property name="text" >
      <string>Save</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>41</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="3" >
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources/resources.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>LaAssemblageConversionBase</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>433</x>
     <y>371</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>LaAssemblageConversionBase</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>433</x>
     <y>377</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
